actionpack/lib/action_dispatch/journey/formatter.rb

```diff
@@ -71,6 +74,9 @@ def generate(name, options, path_parameters)
           missing_keys = missing_keys(route, parameterized_parts)
           next if missing_keys && !missing_keys.empty?
           params = options.dup.delete_if do |key, _|
```

Comment:
We can remove this `dup` here since we are already doing in the top.

Reply:
Done!